prql-compiler/src/sql/gen_query.rs

```diff
@@ -846,4 +846,46 @@ mod test {
           _expr_0 > 3
         "###);
     }
+
+    #[test]
```

Comment:
Generally we've put the standard "does this PRQL create this SQL" in `test.rs`, though looks like we have a few others here. Unless you have a preference, would it be OK to move to `test.rs`?

Reply:
yeah I couldn't find the test.rs at the first place. Will move these cases to test.rs.

Reply:
updated